spyglass/data_extractor/formation.py

```diff
@@ -313,13 +313,19 @@ def get_hosts(self, region, rack=None):
         control_hosts = device_api.zones_zone_id_control_nodes_get(zone_id)
         compute_hosts = device_api.zones_zone_id_devices_get(
             zone_id, type='KVM')
-
         hosts_list = []
+        genesis_set = False
```

Comment:
Is it a design rule to take first node as genesis or hardcoded?

Instead of introducing new flag, why cant the first element is taken out of control_host, handle genesis stuff. And then for loop from [1:]

Reply:
Left it for readability sake we can loop fron [1: ] also